storybook/cypress/e2e/action-list.cy.ts

```diff
@@ -3,6 +3,11 @@ describe("ActionList", () => {
     cy.visit("http://localhost:45000/iframe.html?id=core-action-list--default");
   });
 
+  it("should be accessible", () => {
+    cy.injectAxe();
+    cy.checkA11y("dso-action-list");
```

Comment:
Moeten we hier niet selecteren op de class `hydrated`, net zoals we bij screenshots doen? Ik denk dat het component anders te vroeg wordt gecheckt, of heb jij andere ervaringen?

Reply:
Dan krijg je dit: `Error: No elements found for include in page Context`

Om het werkend te krijgen met `.hydrated` heb ik deze constructie gemaakt:
```typescript
cy.get("dso-action-list.hydrated").should("exist").checkA11y("dso-action-list.hydrated");
``` 

Zal ik dat dan overal doorvoeren?

Reply:
Ja, doe maar. Als ik de documentatie op https://github.com/component-driven/cypress-axe goed begrijp wordt er niet op dezelfde manier een retry gedaan zoals Cypress dat normaal wel doet. Het zal mij niet verbazen als je overal gaat testen op `.hydrated` dat er veel meer accessibility tests gaan falen.

Wat je ook mag proberen is het `cy.checkA11y()` command te overwriten. Maar het lukte mij niet om een niet-standaard Cypres command te overwriten. En anders een custom command toevoegen: https://docs.cypress.io/api/cypress-api/custom-commands.

Dan kun je tenminste de logica centraliseren.

Reply:
We gaan kijken